### DISCUSSION TITLE
Earth to GitHub Community 🌏   [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#53413] [New to GitHub]
Earth Day is this Saturday, April 22, and now more than ever it is important to take care of the planet we call home 🏡. Luckily you can do this while sharpening your programming skills and we have some opportunities below. But first, let’s thank some fellow earthlings for their work in the community recently 👷. 

🌟**Community Superstars**: A round of applause for these community citizens 👏 
* @neutrinomuon: Provided troubleshooting for GitHub Desktop - I appreciated the compassion they showed in their [response](https://github.com/orgs/community/discussions/52375) as well. 
* @Sleavely: They [shared an inconsistency](https://github.com/orgs/community/discussions/52929) about the price of our plans 💰, turns out this was a bug and we were able to get it fixed thanks to their report. 
* @kenf1: For quickly answering a [question](https://github.com/orgs/community/discussions/53059) on acceptable uses for Pages and including a link to our docs. We 💗 it! 

🌍 **Earth Day**: Say thanks to mother nature and contribute to open source  ♻️
* **ClimateHacks**: Work together to save our ecosystem by creating hacks that tackle one of the most pressing issues of our time: climate change 🌞. The Hackathon starts April 21st and you can sign up [here](https://organize.mlh.io/participants/events/9352-climatehacks). 
* **Electricity Maps**: This project aims to provide a free, open-source, and transparent visualization of the carbon intensity of electricity consumption around the world ⚡. [Contribute to the project](https://github.com/electricitymaps/electricitymaps-contrib). 
* **Cloud Carbon Footprint ☁️**: An application that estimates the energy (kilowatt hours) and carbon emissions (metric tons CO2e) of public cloud provider utilization. [Learn how to contribute](https://github.com/cloud-carbon-footprint/cloud-carbon-footprint/blob/trunk/CONTRIBUTING.md). 

🆒 **Cool stuff**™️ happening around GitHub and the Community
* **Redesigned navigation available in Public Beta**: Our navigation has a new look and improved user experience.  [Learn how you can access the beta version.](https://github.blog/changelog/2023-04-05-redesigned-navigation-available-in-public-beta/) 
* **A Beginner’s Guide to Prompt Engineering with GitHub Copilot**: @blackgirlbytes helps you navigate the friendly Copilot skies by [sharing](https://dev.to/github/a-beginners-guide-to-prompt-engineering-with-github-copilot-3ibp) how she improved her communication methods with GitHub Copilot by providing context, examples, and clear instructions in the form of comments and code 🧑‍✈️. 
* **Marking a threaded comment as the answer**: All replies will now have a button to allow the questioner to mark them as the answer ✅. [View the changelog](https://github.blog/changelog/2023-04-17-marking-a-threaded-comment-as-the-answer/) or see how it’s [already been used](https://github.com/orgs/community/discussions/3239#discussioncomment-5638625) in the community. 

Have you seen any more open source projects or hackathons that are helping the planet? Share them below!

**Comment by HammaSolutions** [comment]
im new!

**Reply by queenofcorgis** [reply]
Welcome to GitHub and the GitHub Community @HammaSolutions  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!

**Reply by Ayomide-Philip** [reply]
Can u teach me how to use this

**Comment by neutrinomuon** [comment]
Hi @queenofcorgis, thank you so much for your kind words and for mentioning me in your post :)! It was my pleasure to assist the community, and I'm grateful for the opportunity of helping.

As you said, we celebrate Earth Day today, let's take a moment to reflect on the beauty and wonder of our planet 🌎

**Reply by quanboy247** [reply]
> Hi @queenofcorgis, thank you so much for your kind words and for mentioning me in your post :)! It was my pleasure to assist the community, and I'm grateful for the opportunity of helping.
> 
> 
> 
> As you said, we celebrate Earth Day today, let's take a moment to reflect on the beauty and wonder of our planet 🌎

Welcome.

**Reply by queenofcorgis** [reply]
@neutrinomuon - thank you for being part of the community, so glad to have you helping in the community. Hope you had a great Earth Day!

**Comment by Ayomide-Philip** [comment]
I'm new

**Reply by HammaSolutions** [reply]
im new and i got flagged :( whyyy loan falcon!!
